dplutils/pipeline/ray.py

```diff
@@ -135,6 +135,25 @@ class RemoteTracker:
 
 
 class RayStreamGraphExecutor(StreamingGraphExecutor):
+    """Ray-based implementation of stream graph executor.
+
+    All task outputs are kept in object store and only de-serialized as needed
+    for execution, until yeilded by :meth:`run`, where they are de-serialized on
+    the driver.
+
+    This executor will attempt to pack the cluster, irrespective of any other
```

Comment:
Should somewhere mention that we not just packing but overcommitting?

Reply:
I don't think that is necessary, it utilizes the ray scheduler to make sure tasks are run. The mention above is just to say that it does not consider any other workloads.